core/pom.xml

```diff
@@ -179,6 +179,12 @@ THE SOFTWARE.
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-library</artifactId>
```

Comment:
Should not be necessary—this is a dependency of `junit` IIRC. (Or is that just `hamcrest-core`?) Maybe you actually meant to exclude this dep from some other artifact?

Reply:
Nope, this library appears to not be bundled in JUnit. I was surprised as well